### DISCUSSION TITLE
Fallback on Web for Expo Universal Apps

### DISCUSSION THREAD

**Discussion by mvolonnino** [lodev09/react-native-true-sheet#87] [Ideas]
LOVE this library and the iOS native modal effect where we get that smooth background transforming effect (shrinking). That alone makes me want to utilize this and only this for my bottom sheet / modal needs only.

One show stopper though on universal apps (react native + web) is currently this breaks web within Expo managed. I do not have to much experience with importing truly native components within packages like this so maybe this is something that is very hard to fix - but would be cool if there was a fallback to just a regular react native modal or something like that!

Anyone with more expertise please weigh in or point me in the right direction! 🤝

![Screenshot 2024-08-22 at 9 00 01 AM](https://github.com/user-attachments/assets/8f938579-fe70-4585-9614-db435ab7cd6b)

**Comment by lodev09** [comment]
I wanted to do web but got lazy. Need help 😅

**Comment by lodev09** [comment]
Hopefully someone can contribute and implement a polyfill if it's web -- that looks like a build error. Can utilize other modals in web in-place of TrueSheet

**Comment by lodev09** [comment]
Providing web support for this library is probably not good idea. Sheet implementation is subjective and TrueSheet is designed to be native. I'd recommend creating a wrapper for web with a sheet of your choosing.
